src/AppInstallerRepositoryCore/Microsoft/PredefinedInstalledSourceFactory.h

```diff
@@ -34,6 +34,8 @@ namespace AppInstaller::Repository::Microsoft
             User,
             // Contains machine ARP and machine MSIX
             Machine,
+            // Same as None but creating the source reference causes the next Open to always update the cache
+            NoneWithForcedCacheUpdate,

```

Comment:
Could we ever want to use anything else with forced cache update? If not, can you add a comment why not?